### ISSUE TITLE
Outdated link in Readme

### ISSUE THREAD

**Issue by pprw** [Daniel-KM/Omeka-S-module-Reference#15]
In installation section the link provided leads to outdated Reference-3.4.21 release in gitlab.

> Download the last release [Reference.zip](https://gitlab.com/Daniel-KM/Omeka-S-module-Reference/-/releases) from the list of releases, and uncompress it in the modules directory.

**Comment by Daniel-KM** [comment]
Oui, les zip sont un peu plus compliqués à créer automatiquement sur gitlab.
